### PR TITLE
10 make a navigation bar

### PR DESCRIPTION
**Pull request by Patrick7321**

### CODE REVIEW

File: keyform/templates/keyform/base.html [hunk 1]
```diff
@@ -20,9 +20,15 @@
     <body>
         {% block body %}
             {% block header %}
+            <div class = "main-menu">
                 <div class="title-bar">
+                    <img class="headerImage" src="{% static 'keyform/logo.jpg' %}"/>
                     <div class="title-bar-title">{% trans 'Keyform' %}</div>
+                    <ul>
+                        <li><a href="{% url 'home' %}">Home</a></li>
```
Comment (enki211) [inline keyform/templates/keyform/base.html]: trans tag

File: keyform/static/keyform/css/base.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+.title-bar a:hover {
+    color: #afafaf;
+    cursor: pointer;
+}
+
+.title-bar a {
+    color: white;
+}
+
+span.divider {
```
Comment (enki211) [inline keyform/static/keyform/css/base.css]: This does not seem to be used anywhere

File: keyform/static/keyform/css/base.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+.title-bar li {
+    margin: .75em;
+    display: inline;
+    font-size: 1.5em;
+    vertical-align: middle;
+}
+
+
```
Comment (enki211) [inline keyform/static/keyform/css/base.css]: Too many newlines

**Comment by enki211** [comment]
Please update with master

**Comment by enki211** [comment]
Looks good

File: keyform/templates/keyform/base.html [hunk 4]
```diff
@@ -20,9 +20,15 @@
     <body>
         {% block body %}
             {% block header %}
+            <div class = "main-menu">
```
Comment (shaneallgeier) [inline keyform/templates/keyform/base.html]: Bad spacing around the '=' sign

File: keyform/static/keyform/css/base.css [hunk 5]
```diff
@@ -1,4 +1,39 @@
-ul{
+.title-bar {
+    background-color: #74020C;
```
Comment (enki211) [inline keyform/static/keyform/css/base.css]: If you are going for the MSU color I think it is `#5E0009`

File: keyform/templates/keyform/base.html [hunk 6]
```diff
@@ -20,9 +20,15 @@
...
                 <div class="title-bar">
+                    <img class="headerImage" src="{% static 'keyform/logo.jpg' %}"/>
```
Comment (enki211) [inline keyform/templates/keyform/base.html]: `header-image`

File: keyform/static/keyform/css/base.css (hunk 5 above)
Comment (Patrick7321) [inline keyform/static/keyform/css/base.css]: Well, at this point, I just need it to match the logo that I have

**Pull request merged**